frontend/packages/ceph-storage-plugin/src/components/modals/disk-replacement-modal.tsx

```diff
@@ -108,8 +109,21 @@ const DiskReplacementAction = (props: DiskReplacementActionProps) => {
     event.preventDefault();
     setProgress(true);
     try {
-      const { osd } = alertsMap[diskName];
-      const replacementStatus = replacementMap[diskName]?.status;
+      const { osd } = alertsMap?.[diskName];
+      const {
+        disk: { id, serial, path, node },
+        status: replacementStatus,
```

Comment:
is this replacementStatus came from the index? i can't find declaration here

Reply:
I have updated this. Please check now.